pkg/controller/controller.go

```diff
@@ -80,18 +81,18 @@ func AddToManager(m manager.Manager) error {
 			}
 		case configv1.GCPPlatformType:
 			log.Info("initializing GCP actuator")
-			if plat.GCP == nil {
+			if infraStatus.PlatformStatus == nil || infraStatus.PlatformStatus.GCP == nil {
```

Comment:
for GCP platformstatus will always be set. since it's a 4.2 platform

Reply:
ack... which is why if it's not there we should just exit

Reply:
+1